src/pages/gen2/deploy-and-host/fullstack-branching/cross-account-deployments/index.mdx

```diff
@@ -0,0 +1,169 @@
+export const meta = {
+  title: 'Cross account deployments',
+  description:
+    'Setup a cross account deployment pipeline powered by Amazon CodeCatalyst and AWS Amplify Hosting.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+While building with Amplify CI/CD offers benefits such as zero-config setup, fullstack previews, centralized secrets management, Amplify Gen 2 makes it easy to integrate fullstack CI/CD into your custom pipelines (e.g. AWS CodePipeline, Amazon CodeCatalyst, GitHub Actions etc.). This guide will walk you through how to create a customized deployment pipeline for your AWS Amplify Gen 2 applications powered by Amazon CodeCatalyst and AWS Amplify Hosting. You can follow along the step-by-step process to build this deployment workflow for your cross account setup and can modify it based on your specific project needs.
```

Comment:
This guide walks you through how to create a trunk-based, multi-region deployment pipeline for your AWS Amplify Gen 2 applications. In this tutorial, we will be using Amazon CodeCatalyst and AWS Amplify Hosting, but you can choose any CI/CD provider.

Reply:
Done, the introduction has been updated.